src/styles/syntaxHighlight.css

```
@@ -19,5 +19,5 @@ div[data-rehype-pretty-code-fragment] pre code .line {
 }
 
 div[data-rehype-pretty-code-fragment] .highlighted {
-  @apply bg-green-600/20
+  @apply bg-green-600/20 w-[800px] 
```

I think I will just hide it until I figure this one out
```suggestion
  @apply sm:bg-green-600/20 
```